app/scripts/controllers/eventDetails.js

```diff
@@ -672,14 +672,25 @@ angular
         const currentPurpose =
           $scope.ministryPurposes &&
           $scope.ministryPurposes.find((m) => m.id === $scope.conference.type);
-        return currentMinistry.eventTypes &&
+        return currentMinistry &&
```

Comment:
You can use optional chaining for `currentMinistry?.eventTypes` since you're not using `.includes()` on it.

```suggestion
        return currentMinistry?.eventTypes &&
```